app/src/main/res/menu/bottom_menu.xml

```diff
@@ -2,15 +2,15 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
 
     <item
-        android:id="@+id/menu_notes"
+        android:id="@+id/notesFragment"
```

Comment:
Generally we don't wanna reference fragment here because xml defines the view (UI logic), which has nothing to do with the fragment (implementation logic) itself. I like the names you had before better.

Reply:
not 100% but pretty sure with the new implementation it auto links to fragments so I have to name it the same as the specific fragment class name